src/Chainweb/Pact/Types.hs

```diff
@@ -503,15 +517,32 @@ pactLoggers logger = P.Loggers $ P.mkLogger (error "ignored") fun def
         let namedLogger = addLabel ("logger", pack n) logger
         logFunctionText namedLogger (pactLogLevel cat) $ pack msg
 
+-- | Write log message
+--
+logg_ :: MonadIO m => P.Logger -> String -> String -> m ()
```

Comment:
Instead of the alternates, why not just migrate to this `MonadIO` version?

Reply:
Most uses are within `PactServiceM c a`. In that context, using the `MonadIO` version would require to first extract the logger from the pact environment via `view psLogger`, which seems inconvenient.